core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -646,4 +646,13 @@ package object config {
       .stringConf
       .toSequence
       .createWithDefault(Nil)
+
+  private[spark] val EXECUTOR_HEARTBEAT_INTERVAL =
```

Comment:
This change wasn't generally back-ported to 2.4, so I don't think we should add one field of it.

Reply:
removed